core-services/image-mirroring/knative/mapping_knative_v0_9_quay

```diff
@@ -17,4 +17,16 @@ registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-ingress
 registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-pong quay.io/openshift-knative/knative-eventing-pong:v0.9.0
 registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sendevent quay.io/openshift-knative/knative-eventing-sendevent:v0.9.0
 registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-webhook quay.io/openshift-knative/knative-eventing-webhook:v0.9.0
-registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sources-controller quay.io/openshift-knative/knative-eventing-sources-controller:v0.9.0
\ No newline at end of file
+registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sources-controller quay.io/openshift-knative/knative-eventing-sources-controller:v0.9.0
+registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sources-kafka-source-adapter quay.io/openshift-knative/knative-eventing-sources-kafka-source-adapter:v0.9.0
+registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sources-kafka-source-controller quay.io/openshift-knative/knative-eventing-sources-kafka-source-controller:v0.9.0
+registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sources-kafka-channel-controller quay.io/openshift-knative/knative-eventing-sources-kafka-channel-controller:v0.9.0
+registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sources-kafka-channel-dispatcher quay.io/openshift-knative/knative-eventing-sources-kafka-channel-dispatcher:v0.9.0
+registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sources-kafka-channel-webhook quay.io/openshift-knative/knative-eventing-sources-kafka-channel-webhook:v0.9.0
+registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sources-camel-source-controller quay.io/openshift-knative/knative-eventing-sources-camel-source-controller:v0.9.0
+registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sources-github-receive-adapter quay.io/openshift-knative/knative-eventing-sources-github-receive-adapter:v0.9.0
+registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sources-github-source-controller quay.io/openshift-knative/knative-eventing-sources-github-source-controller:v0.9.0
+registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sources-heartbeats quay.io/openshift-knative/knative-eventing-sources-heartbeats:v0.9.0
+registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sources-websocketsource quay.io/openshift-knative/knative-eventing-sources-websocketsource:v0.9.0
+registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sources-event_display quay.io/openshift-knative/knative-eventing-sources-event_display:v0.9.0
```

Comment:
that image does not exist.  CI server requires that we do not have `_` in the name ( no idea why).

When we generate the CI config, we already "catch" that, see:
https://github.com/openshift/knative-eventing/blob/master/openshift/ci-operator/generate-ci-config.sh#L50

@alanconway I guess that's needed when generating this file too :-) 

see that `docker pull registry.svc.ci.openshift.org/openshift/knative-v0.9.0:knative-eventing-sources-event-display` works

Reply:
Fixed, is this the final touch?